lab_5_scrapper/scrapper.py

```diff
@@ -127,11 +233,27 @@ def _extract_url(self, article_bs: BeautifulSoup) -> str:
         Returns:
             str: Url from HTML
         """
+        link_tags = article_bs.find_all('a', class_='listing-preview__content')
+        for link in link_tags:
+            url = link.get('href')
+            if url:
+                return url
+            return ''
```

Comment:
you can specify default return value for get, then it will help to avoid this awkward return construction

Reply:
Changed it